src/components/Layout/Header/NavBar/BuySell.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, useMediaQuery } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { BuySellIcon } from 'components/Icons/Buysell'
+import { Text } from 'components/Text'
+
+import { MainNavLink } from './MainNavLink'
+
+export const BuySellHeader = () => {
+  const [isLargerThan765] = useMediaQuery('(min-width: 765px)')
+  const translate = useTranslate()
+
+  return (
+    <>
+      {isLargerThan765 && (
```

Comment:
if we're doing this media query thing do this in separate return statements rather than nesting